matlab/post_processing/loadMCResults.m

```diff
@@ -409,7 +409,27 @@
                                                        - imag(ROfFxAndAngle.Mean) .* imag(ROfFxAndAngle.Mean)) / json.N);
             end
             results{di}.ROfFxAndAngle = ROfFxAndAngle;
-        case 'TDiffuse'
+        case 'ROfFxAndMaxDepth'
+            ROfFxAndMaxDepth.Name = detector.Name;
+            tempFx = detector.Fx;
+            tempMaxDepth = detector.MaxDepth;
+            ROfFxAndMaxDepth.Fx = linspace((tempFx.Start), (tempFx.Stop), (tempFx.Count));
+            ROfFxAndMaxDepth.MaxDepth = linspace((tempMaxDepth.Start), (tempMaxDepth.Stop), (tempMaxDepth.Count));
+            ROfFxAndMaxDepth.Fx_Midpoints = ROfFxAndMaxDepth.Fx;
+            ROfFxAndMaxDepth.MaxDepth_Midpoints = (ROfFxAndMaxDepth.MaxDepth(1:end-1) + ROfFxAndMaxDepth.MaxDepth(2:end))/2;
+            tempData = readBinaryData([datadir slash detector.Name],[2*(length(ROfFxAndMaxDepth.MaxDepth)-1), length(ROfFxAndMaxDepth.Fx)]); % column major but complex
+            ROfFxAndMaxDepth.Mean = tempData(1:2:end,:) + 1i*tempData(2:2:end,:);  
+            ROfFxAndMaxDepth.Amplitude = abs(ROfFxAndMaxDepth.Mean);
+            ROfFxAndMaxDepth.Phase = -angle(ROfFxAndMaxDepth.Mean);
+            if(detector.TallySecondMoment && exist([datadir slash detector.Name '_2'],'file'))
+                tempData = readBinaryData([datadir slash detector.Name '_2'],[2*(length(ROfFxAndMaxDepth.MaxDepth)-1), length(ROfFxAndMaxDepth.Fx)]);
+                ROfFxAndMaxDepth.SecondMoment = tempData(1:2:end,:) + tempData(2:2:end,:); % SecondMoment=E[re^2]+E[im^2] is real
+                % SD=sqrt( SecondMoment - E[re]^2 - E[im]^2 )
+                ROfFxAndMaxDepth.Stdev = sqrt((ROfFxAndMaxDepth.SecondMoment - real(ROfFxAndMaxDepth.Mean) .* real(ROfFxAndMaxDepth.Mean) ...
+                                                       - imag(ROfFxAndMaxDepth.Mean) .* imag(ROfFxAndMaxDepth.Mean)) / json.N);

```

Comment:
@hayakawa16, there is a strange character in this file, it may not be an issue but I noticed it so I wanted to mention it.

Reply:
Oh, that's the carriage return or line feed character.  I ran that file so I know it works.  I will correct.

Reply:
How did you view that character?  I cannot see it using Notepad or MATLAB.  And when I put the cursor after the third dot, the cursor cannot move to the right.

Reply:
I can see it when I look at the changes here in GitHub, in the files tab.